worlds/sc2/docs/setup_fr.md

```diff
@@ -0,0 +1,191 @@
+# Guide d'installation du *StarCraft 2 Randomizer*
+
+Ce guide contient les instructions pour installer et dépanner le client de *StarCraft 2 Archipelago*, ainsi que des 
+indications pour obtenir un fichier de configuration de *Starcraft 2 Archipelago* et comment modifier ce dernier.
+
+## Logiciels requis
+
+- [*StarCraft 2*](https://starcraft2.com/en-us/)
+- [La version la plus récente d'Archipelago](https://github.com/ArchipelagoMW/Archipelago/releases)
+
+## Comment est-ce que j'installe ce *randomizer*?
```

Comment:
randomizer ou randomiseur?

Reply:
Je sais que vous avez mis cela en gras alors ce n'est pas important, mais pour un peu de "consistency".

Reply:
I like the idea of keeping the English word since I feel that this terms has more meaning in our context and that translating it would be akin to translate "Starcraft 2". Is that acceptable?

Reply:
Yep!